tracer/src/Datadog.Trace/Iast/Aspects/System.Text.Json/JsonDocumentAspects.cs

```diff
@@ -5,7 +5,7 @@
 
 #nullable enable
 
-#if NETCOREAPP
```

Comment:
I don't understand why here you change this to `NETCOREAPP3_0_OR_GREATER` where in other parts you are changing it back to `NETCOREAPP`?

Reply:
Because, AFAICT, these _aren't_ supported on .NET Core 2.1 😄